catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -396,4 +402,9 @@ private Serializable parseDefaultValue(AttributeDescriptor descriptor, String de
             List<String> metacardTypes;
         }
     }
+
+    BundleContext getBundleContext() {
```

Comment:
need to refactor eveywhere FrameworkUtil is called in t his class to use this method instead